formats/html_publication/publisher/details.json

```diff
@@ -21,6 +21,12 @@
     },
     "first_published_version": {
       "type": "boolean"
+    },
+    "isbn": {
+      "type": "string"
+    },
+    "print_meta_data_contact_address": {
```

Comment:
I don't think this needs to have `print_` in the name, it may not always be used for print.
I'd note it's use in a description field.

Should the ISBN and contact address be displayed on screen for users as well as those printing?